### DISCUSSION TITLE
Odbc multiple result sets

### DISCUSSION THREAD

**Discussion by Aarstad** [pocoproject/poco#3581] [General]
I've seen it mentioned a few times in stack overflow and here. It's said to be on the roadmap but not implemented yet. 

Where would one start implementing this? Is it hard? Doesn't odbc allow for multiple result sets?

**Comment by aleks-f** [comment]
It already exists:

https://github.com/pocoproject/poco/blob/poco-1.11.2/Data/ODBC/testsuite/src/SQLExecutor.h#L494-L497
https://github.com/pocoproject/poco/blob/poco-1.11.2/Data/ODBC/testsuite/src/SQLExecutor.cpp#L3422-L3451

**Reply by xakod** [reply]
How to use this if returned value will be null and i don't wanna provide default value? Is there any way to use RecordSet? 
I tried RecordSet but it returned only first result set

**Reply by aleks-f** [reply]
No, `RecordSet` does not support that functionality
